app/js/controllers/main.js

```diff
@@ -35,11 +53,82 @@ export default class MainController extends Controller {
     };
   }
 
+  _lazyLoadModules() {
+    return new Promise((resolve) => {
+      if (this._loadedModules) {
+        resolve();
+        return;
+      }
+
+      var editView = new EditView();
+      var actionMenuView = new ActionMenuView();
+      var settingsView = new SettingsView();
+      var viewSourceView = new ViewSourceView();
+      var appendChildView = new AppendChildView();
+      var moveView = new MoveView();
+      var mainView = new MainView({
```

Comment:
I really don't like how gluey this has become. You're just moving code around, so I'm not going to make it an action item, but we should really think about this. We originally used this pattern of passing controllers and views into themselves and each other because we wanted to be able to make views interchangeable within controllers. But we've only needed that in one case with the Sharing app, and never that I know of within the Customizer. So I'd say that this pattern optimizes for a use case that is rare for us. Maybe we should more deeply link controllers and views, e.g. controllers initialize their own view.